.pre-commit-config.yaml

```diff
@@ -22,8 +22,13 @@ repos:
     - id: rst-backticks
     - id: rst-directive-colons
     - id: rst-inline-touching-normal
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.19.0
+    hooks:
+      - id: pyupgrade
+        args: [--py39-plus]
```

Comment:
Python 3.8 is no longer supported, so it makes sense to use at least Python 3.9:
http://endoflife.date/python